<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListItemBox</class>
 <widget class="QWidget" name="ListItemBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="musicNameBox" native="true">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="likeBtn">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#likeBtn
{
 border:none;

}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="musicNameLabel">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>超级敏感</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="VIPLabel">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#VIPLabel
{
 border:1px solid #1ECD96;
 color:#1ECD96;
 border-radius:2px;
}</string>
        </property>
        <property name="text">
         <string>VIP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SQLabel">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#SQLabel
{
  border:1px solid #FF6600;
  color:#FF6600;
  border-radius:2px;
 }</string>
        </property>
        <property name="text">
         <string>SQ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>102</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>musicNameLabel</zorder>
     <zorder>VIPLabel</zorder>
     <zorder>SQLabel</zorder>
     <zorder>likeBtn</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="musicSingerBox" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="musicSingerLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#musicSingerNameLabel
{
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string>A-Soul</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="musicAlbumBox" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="musicAlbumLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>超级敏感</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
